src/index.js

```diff
@@ -47,6 +47,10 @@ class HCaptcha extends React.Component {
     }
 
     componentDidMount () { // Once captcha is mounted intialize hCaptcha - hCaptcha
+      if (this._hcaptcha) {
+        return;
+      }
```

Comment:
Are we sure this is correct? This check only validates whether the API has loaded within the current window context.

Reply:
I also thought if it's safer to use state.captchaId
in general, this logic works as expected, as this._hcaptcha is set only in componentDidMount, so renderCaptcha is still called

Reply:
it looks as the safe solution to me as it's the way to ensure that componentDidMount is called only once
we could actually use any other variable for that
also React.StrictMode calls componentDidMount twice only on dev mode, so it should be safe especially on prod

however, as the alternative, we could go with this PR: https://github.com/hCaptcha/react-hcaptcha/pull/251
please take a look

@brdlyptrs @vladislav-nick-imi @damian-rodriguez-imi